docs/developer/eventing/sources/kafka-source/README.md

```diff
@@ -285,6 +285,31 @@ To specify it, add the label `kafkasources.sources.knative.dev/key-type` to the
         name: event-display
    ```
 
+## (Optional) Specify the initial offset
+
+By default the `KafkaSource` starts consuming from the `latest` offset in each partition. In case, you want to consume from the earliest offset, use this field to configure the same.
```

Comment:
`to configure the same` does not sound right. What about `In case you want to consume from the earliest offset, set the initialOffset field to earliest`?